langfuse/_task_manager/ingestion_consumer.py

```diff
@@ -1,56 +1,45 @@
-"""@private"""
-
-import atexit
 import json
 import logging
-import queue
 import threading
-from queue import Empty, Queue
 import time
-from typing import List, Any, Optional
-import typing
 
-from langfuse.Sampler import Sampler
-from langfuse.parse_error import handle_exception
-from langfuse.request import APIError
-from langfuse.utils import _get_timestamp
-from langfuse.types import MaskFunction
+from queue import Empty, Queue
+from typing import Any, List, Optional
+
+import backoff
 
 try:
-    import pydantic.v1 as pydantic  # type: ignore
+    import pydantic.v1 as pydantic
 except ImportError:
-    import pydantic  # type: ignore
-
-
-import backoff
+    import pydantic
 
-from langfuse.request import LangfuseClient
+from langfuse.api.client import FernLangfuse
+from langfuse.parse_error import handle_exception
+from langfuse.request import APIError, LangfuseClient
+from langfuse.Sampler import Sampler
 from langfuse.serializer import EventSerializer
+from langfuse.types import MaskFunction
 
-# largest message size in db is 331_000 bytes right now
-MAX_MSG_SIZE = 1_000_000
-
-# https://vercel.com/docs/functions/serverless-functions/runtimes#request-body-size
-# The maximum payload size for the request body or the response body of a Serverless Function is 4.5 MB
-# 4_500_000 Bytes = 4.5 MB
-# configured to be 3 MB to be safe
+from .media_manager import MediaManager
 
-BATCH_SIZE_LIMIT = 2_500_000
+MAX_EVENT_SIZE_BYTES = 1_000_000
+MAX_BATCH_SIZE_BYTES = 2_500_000
 
 
-class LangfuseMetadata(pydantic.BaseModel):
+class IngestionMetadata(pydantic.BaseModel):
     batch_size: int
-    sdk_integration: typing.Optional[str] = None
-    sdk_name: str = None
-    sdk_version: str = None
-    public_key: str = None
+    sdk_integration: Optional[str] = None
+    sdk_name: Optional[str] = None
+    sdk_version: Optional[str] = None
+    public_key: Optional[str] = None
```

Comment:
why are these optional?

Reply:
they are required, changed that, thanks!